Content.Server/PDA/PDASystem.cs

```diff
@@ -188,14 +260,63 @@ private void AfterUIOpen(EntityUid uid, PDAComponent pda, AfterActivatableUIOpen
             if (!_uiSystem.TryGetUi(pda.Owner, PDAUiKey.Key, out var ui))
                 return;
 
-            var ownerInfo = new PDAIdInfoText
+            var ownerInfo = new PdaIdInfoText
             {
                 ActualOwnerName = pda.OwnerName,
                 IdOwner = pda.ContainedID?.FullName,
                 JobTitle = pda.ContainedID?.JobTitle
             };
 
-            var state = new PDAUpdateState(pda.FlashlightOn, pda.PenSlot.HasItem, ownerInfo, pda.StationName, true, HasComp<InstrumentComponent>(pda.Owner), GetDeviceNetAddress(pda.Owner));
+            var currentTime = GameTicker.RoundDuration();
+            var stationTime = new StationTimeText
+            {
+                Hours = currentTime.Hours.ToString(),
+                Minutes = currentTime.Minutes.ToString(),
+            };
+
+            var stationAlert = new StationAlert
+            {
+                Level = null,
+                Color = Color.White
+            };
+
+            var stationUid = _stationSystem.GetOwningStation(pda.Owner);
+            if (TryComp(stationUid, out AlertLevelComponent? alertComp) &&
```

Comment:
```suggestion
            var stationUid = _stationSystem.GetOwningStation(uid);
            if (TryComp(stationUid, out AlertLevelComponent? alertComp) &&
```

.Owner is deprecated.